### ISSUE TITLE
hello, i want to ask the code of run_LNLTC is not container dataset classifier(Ld)? and the result of finnal PT is the paper's "LNPL w/o Rel"?

### ISSUE THREAD

**Issue by li161** [amurtadha/LNPL#6]

**Comment by amurtadha** [comment]
run_LNLTC includes l_d in lines 656
                loss_rev = criterion_noise(logits_rev, evid)
the result of finale PT is LNPL (ALL), It first train with NN and L_d (i.e., _train_nt), second it refine some labels for the final PT training. 
So the finale PT is trained after refining labels resulting from _train_nt
